lib/pipeline.js

```diff
@@ -189,6 +189,37 @@ Pipeline.prototype = {
   },
 
   /**
+   * Insert given streams into the pipeline at a given index
+   * @param {Number} index
+   * @param {...Stream} streams
+   * @returns {Stream[]} inserted
+   */
+  insert: function( index ) {
+
+    if( typeof index !== 'number' ) {
```

Comment:
Not strictly related to this PR (sorry) but could we add jslint to this repo, so that the code is formatted the same as e.g. https://github.com/resin-io/etcher ?